app/src/main/res/values/styles.xml

```diff
@@ -459,4 +459,9 @@
     <item name="android:gravity">center_horizontal</item>
   </style>
 
+  <!-- ShapeableImageView -->
+  <style name="ShapeAppearanceOverlay.App.Circular" parent="">
+    <item name="cornerSize">50%</item>
```

Comment:
I followed this documentation
[https://howtodoandroid.com/shapeableimageview-material-components-android/](url)

Reply:
[https://howtodoandroid.com/shapeableimageview-material-components-android/](url)
I followed the above documentation

Reply:
https://howtodoandroid.com/shapeableimageview-material-components-android/

Reply:
Given other styles names that we have like
`TextInputLayout`,
`TextInputEditText`,
`AppTabTextAppearance`

I think `ShapeableImageView.Circular` can be a good name because it clearly mentions where to use this theme (ShapeableImageView) and also what will be the output of the theme (Circular).